prefect_saturn/core.py

```diff
@@ -105,10 +107,17 @@ def _hash_flow(self, flow: Flow) -> str:
         flow with a given name, in a given Prefect Cloud project, for a given
         Prefect Cloud tenant.
         """
+        prefect_version = Version(version("prefect"))
+
+        if parse("0.13.0") <= prefect_version < parse("0.15.0"):
+            tenant_id = Client()._active_tenant_id  # type: ignore # pylint: disable=no-member
+        elif prefect_version >= parse("0.15.0"):
```

Comment:
I think this achieves the same:

```suggestion
        if prefect_version < parse("0.15.0"):
            tenant_id = Client()._active_tenant_id  # type: ignore # pylint: disable=no-member
        else:
```

Reply:
This won't work for every Prefect version under `0.15.0`, but we do require `>0.13.0` anyway so no big deal.
I doubt we will come back and support earlier versions of Prefect, and if we need to do something different for `0.16.0` we can adjust this code in the future. Adjusting.

Reply:
Simplified the if statement in https://github.com/saturncloud/prefect-saturn/pull/47/commits/a969323b0dcddfbee6faa656ecd3b4d46b815408